docs/guides/forms/Forms.stories.mdx

```diff
@@ -1,5 +1,79 @@
-import { Meta } from "@storybook/addon-docs";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import * as FormSamples from "./FormsSamples";
 
-<Meta title="Guides/Forms/Main" />
+<Meta title="Guides/Forms/Guide" />
 
 # Forms
+
+Web forms play a crucial role in web applications, as they are the primary form to collect user information.
+By providing a standardised way to capture user input and validat it, we can ensure a consistent user experience across the different applications.
+
+## Form Components
+
+UI Kit provides a set of **Form Components** to build forms. These components have the goals of providing a **consistent React API** to build forms **following the HV Design spec** and
+**ease of use and integration with** native HTML form, React forms and form libraries.
+
+The base API of all the form components is established by the **[Form Element](https://lumada-design.github.io/uikit/master/?path=/docs/guides-forms-form-element--docs)** component.
+It provides the necessary props to listen to the events, manage the component `status` as well as a `value` prop to make each component **controlled**.
+
+The UI Kit library currently offers the following form components:
```

Comment:
I think the Filter Group is missing as a form component if I'm not mistaken.

Reply:
It's not - I'll add a few words about it 👍

Reply:
Okay, my bad! I though it was since it uses the `HvFormElement` 😅